addons/mail/static/src/models/messaging_notification_handler/messaging_notification_handler.js

```diff
@@ -389,6 +404,8 @@ function factory(dependencies) {
                 return this._handleNotificationChannelSeen(data.channel_id, data);
             } else if (type === 'deletion') {
                 return this._handleNotificationPartnerDeletion(data);
+            } else if (type === 'mail_user_settings') {
+                return this._handleNotificationMailUserSettings(data.payload);
```

Comment:
Move the handler in the switch type in the main method `_handleNotifications`

Consider `_handleNotificationPartner` is deprecated

Reply:
refactored